requirements.txt

```diff
@@ -1,4 +1,5 @@
 matplotlib
 numpy
 scipy
-setuptools
\ No newline at end of file
+setuptools
+sklearn
```

Comment:
Technically you do not use Scikit-learn in this PR, but let it be as we will need it later.